CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Javis.jl - Changelog
 
+## v0.4.1
```

Comment:
We normally increase the minor version number when we add dependencies.
I've also changed the base of this to master. I think we don't need to wait any longer to tag v0.5. once this is in :smile: